blocks/columns/columns.css

```
@@ -300,6 +327,24 @@ main .section-bottom .columns-2-cols > div > :nth-child(2) p:nth-of-type(3) {
         font-size: var(--font-size-16);
     }
 
+    .columns.quarter-circle.image-start > div > div:first-child, .columns.quarter-circle.image-end > div > div:last-child  {
```

Move multiple selectors to their own line.
```suggestion
    .columns.quarter-circle.image-start > div > div:first-child, 
    .columns.quarter-circle.image-end > div > div:last-child  {
```